backend/src/models/kanban/SprintStatistics.ts

```diff
@@ -1,5 +1,5 @@
 export class SprintStatistics {
-  public completedCardsNumber?: Array<CompletedCardNumber>;
+  public completedCardNumber?: Array<CompletedCardNumber>;
```

Comment:
`completedCardNumber` seems like just one number , `completedCardNumberList` may be clearer.
And param in constructor is not renamed.